src/util/image.ts

```diff
@@ -0,0 +1,94 @@
+import { saveAs } from 'file-saver';
+
+export enum IMAGE_EXTENSIONS {
+  SVG = '.svg',
+  PNG = '.png',
+  JPEG = '.jpeg',
+}
+
+export const IMAGE_MIME_TYPES = {
+  [IMAGE_EXTENSIONS.SVG]: 'image/svg+xml',
+  [IMAGE_EXTENSIONS.PNG]: 'image/png',
+  [IMAGE_EXTENSIONS.JPEG]: 'image/jpeg',
+};
+
+export const dataURItoBlob = (dataURI) => {
+  // convert base64 to raw binary data held in a string
```

Comment:
do we need try-catch here?

Reply:
I think no, in worst case the error is throws and i catch it in component & show error notification